tests/zenko_e2e/backbeat/ReplicationUtility.js

```diff
@@ -561,6 +563,57 @@ class ReplicationUtility {
             return cb();
         });
     }
+    
+    assertNoBlob(containerName, key, cb) {
+        this.azure.getBlobProperties(containerName, key, err => {
+            assert(err.code);
+            return cb();
+        });
+    }
+
+    checkReplicationStatus(locationName, cb) {
```

Comment:
I like this approach of using helper methods for the test suite. I think this warrants its own utility class since it's querying the backbeat API routes. Perhaps introduce something like `BackbeatAPIUtility`.